docs/guide/algos.rst

```diff
@@ -9,6 +9,7 @@ along with some useful characteristics: support for discrete/continuous actions,
 Name         ``Box``     ``Discrete`` ``MultiDiscrete`` ``MultiBinary`` Multi Processing
 ============ =========== ============ ================= =============== ================
 TQC          ✔️          ❌            ❌                ❌              ❌
+QRDQN        ️❌          ✔            ❌                ❌              ❌
```

Comment:
pleas re-use the same character as above (copy-paste should work) and also update the README ;)

Reply:
You mean, use "QR-DQN", not "QRDQN"?

Reply:
yes and also the ✔️ seems to be different